RELEASING.md

```diff
@@ -0,0 +1,194 @@
+# How to Create a Release of OpenTelemetry Operations Java (for Maintainers Only)
+
+## Build Environments  
+
+We deploy OpenTelemetry Operations Java to Maven Central under Ubuntu 14.04.
+
+Other systems may also work, however the system used for build and deploy must
+be able to run the [mock
+server](https://github.com/googleinterns/cloud-operations-api-mock).
+
+## Prerequisites
+
+### Setup OSSRH and Signing
+
+If you haven't deployed artifacts to Maven Central before, you need to set up
+your OSSRH (OSS Repository Hosting) account and signing keys.
+
+- Follow the instructions on [this
+  page](http://central.sonatype.org/pages/ossrh-guide.html) to set up an account
+  with OSSRH.
+  - You only need to create the account, not set up a new project
+  - Contact an OpenTelemetry Operations Java maintainer to add your account
+        after you have created it.
+- (For release deployment only) [Install
+    GnuPG](http://central.sonatype.org/pages/working-with-pgp-signatures.html#installing-gnupg)
+    and [generate your key
+    pair](http://central.sonatype.org/pages/working-with-pgp-signatures.html#generating-a-key-pair).
+    You'll also need to [publish your public
+    key](http://central.sonatype.org/pages/working-with-pgp-signatures.html#distributing-your-public-key)
+    to make it visible to the Sonatype servers.
+- Put your GnuPG key password and OSSRH account information in
+  `<your-home-directory>/.gradle/gradle.properties`:
+
+    ```text
+    # You need the signing properties only if you are making release deployment
+    signing.keyId=<8-character-public-key-id>
+    signing.password=<key-password>
+    signing.secretKeyRingFile=<your-home-directory>/.gnupg/secring.gpg
+
+    ossrhUsername=<ossrh-username>
+    ossrhPassword=<ossrh-password>
+    checkstyle.ignoreFailures=false
+    ```
+
+## Download the mock server
+
+- Download the [mock server
+  executable](https://github.com/googleinterns/cloud-operations-api-mock/releases),
+  and save the path.
+
+    ```bash
+    $ VERSION=v2-alpha
+    $ BINARY=mock_server-x64-linux-$VERSION
+    $ curl -L -o $BINARY https://github.com/googleinterns/cloud-operations-api-mock/releases/download/$VERSION/$BINARY
+    $ chmod +x $BINARY
+    $ MOCKSERVER=$PWD/$BINARY
```

Comment:
re: suggestion to invoke a script above. You're using this env var below, so we need to keep it.
or we could add this line to the bash script and invoke it as `source ./get_mock_server.sh`